packages/core-snapshots-cli/lib/db/queries/blocks/latest.sql

```diff
@@ -0,0 +1,3 @@
+SELECT *
+FROM BLOCKS
```

Comment:
Properties in queries should be all lower-case, only SQL keywords should be uppercase.